v1.2: Update auto-batching note (#2419)
meilisearch/documentation: learn/core_concepts/documents.mdx

### Auto-batching

<Capsule intent="danger">

> Currently, auto-batching has been disabled for document addition and deletion. See [this GitHub issue](https://github.com/meilisearch/meilisearch/issues/3664) for more details.

Reviewer: Would it be better to say "is disabled" instead of "has been disabled"?

> Currently, auto-batching has been disabled for document addition and deletion. See [this GitHub issue](https://github.com/meilisearch/meilisearch/issues/3664) for more details.

Author: ```suggestion
Currently, auto-batching is disabled for document addition and deletion. See [this GitHub issue](https://github.com/meilisearch/meilisearch/issues/3664) for more details.
```
